### ISSUE TITLE
Review timestamp is displayed as unix time, not formatted

### ISSUE THREAD

**Issue by JavaZauber** [GothenburgBitFactory/taskwarrior#2060]
Hi. I'm using the tasksh review feature and I'm pretty happy with it.

Review timestamp is not formatted and this seems to be a bug.
I first ignored it because I _thought_ it was an internal number just for the tasksh. Looking at the modification log I see that it's formatted, a clear discrepancy

![screenshot of a console displaying the above stated bug, there are gray censor areas visible and thus the screenshot is very boring. Greetings to the screenreader people](https://user-images.githubusercontent.com/6533815/44906438-2d41b700-ad15-11e8-8c28-e58582d3ef3c.png)

```
task 2.5.1
   Platform: Linux      # Windows Subsystem für Linux

Compiler
    Version: 5.4.0 20160609
       Caps: +stdc +stdc_hosted +LP64 +c8 +i32 +l64 +vp64 +time_t64
 Compliance: C++11

Build Features
      CMake: 3.5.1
    libuuid: libuuid + uuid_unparse_lower
  libgnutls: 3.4.10
 Build type: None

Configuration
       File: /home/jonasdralle/.taskrc (found), 607 bytes, mode 100666
       Data: /mnt/c/Users/Jonas.Dralle/.task (found), dir, mode 40777
    Locking: Enabled
         GC: Enabled
     Server:
      Trust: strict
Certificate: , not readable, 0 bytes
        Key: , not readable, 0 bytes
    Ciphers: NORMAL
      Creds:

Hooks
     System: Enabled
   Location: /mnt/c/Users/Jonas.Dralle/.task/hooks
             (-none-)

Tests
      $TERM: xterm-256color (170x36)
       Dups: Scanned 168 tasks for duplicate UUIDs:
             No duplicates found
 Broken ref: Scanned 168 tasks for broken references:
             No broken references found
```

```
$ tasksh --version
1.2.0
```

TL;DR format uda dates, thanks.

**Comment by pbeckingham** [comment]
Good suggestion.

**Comment by tbabej** [comment]
Reproduced on 2.6.0.

**Comment by tbabej** [comment]
Fixed in 2.6.0.
